aviary/docs/getting_started/installation.md

```diff
@@ -28,7 +20,11 @@ This command should be performed while in the folder containing ``setup.py``.
 
 ```{note}
 You can do this editable installation with any of the `[test]` or `[all]` options as well.
-``````
+```
+
+```{note}
+You can install the optional package [pyOptSparse by following the instructions here](https://mdolab-pyoptsparse.readthedocs-hosted.com/en/latest/install.html). If you do not need the SNOPT optimizer, installing pyOptSparse is as simple as running `conda install -c conda-forge pyoptsparse`.
+```
```

Comment:
duplicate of below?

Reply:
Yes though it's on purpose; I've had multiple questions about how to install pyOptSparse so I really want to drill that in. Also the directions below are advertised for "developers on Linux" so I wanted to also have this info for a wider audience.